src/core/dns/transact.js

```diff
@@ -117,7 +117,6 @@ export class TcpTx {
       sb.qlenBufOffset = bufutil.recycleBuffer(sb.qlenBuf);
       sb.qBuf = null;
       sb.qBufOffset = 0;
-      return;
```

Comment:
this `return` is to avoid accidental code execution when something else is added beyond the current `if-else` block... That is to say, the `return` here exists for humans even if the interpreter / linter might think it is superfluous.